sh/build-database.sh

```diff
@@ -16,38 +16,38 @@ set +a
 TIMEOUT=${BUILD_TIMEOUT:-8}
 
 # build the test database
-mysql -u $DB_USER -p$DB_PASS -e "DROP DATABASE IF EXISTS $DB_NAME ;" &> /dev/null
```

Comment:
Why don't we redirect the output to `/dev/null` now ?

Reply:
Ah - this was a feature requested by @jmcameron.  We basically eliminate all ability to debug by sending all errors into `/dev/null/`.  I figured this was a bit too much.  Do you think we should put it back?

Reply:
Can we set up the script to have an argument that determines whether the output goes to /dev/null or to regular output?

Reply:
If you want, I can take a cut at setting up the script with an optional argument for debugging:

  sh/build-database.sh debug

would force output to output to go to stdout (if I can figure out how to do that...)

-Jonathan C

Reply:
Got a test bash script working for the debug option here.

Reply:
[testo.txt](https://github.com/IMA-WorldHealth/bhima-2.X/files/2336072/testo.txt)

Download, remove .txt and chmod +x and try it with:
   ./testo
   ./testo debug